index.html

```diff
@@ -1,28 +1,30 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Car Assistant</title>
-    <script src="libs/angular/angular.min.js"></script>
-    <script src="libs/angular/angular-route.min.js"></script>
-    <script src="libs/angular/angular-resource.min.js"></script>
-    <script src="libs/bootstrap/js/bootstrap.min.js"></script>
-    <script src="libs/ionic/js/ionic.bundle.js"></script>
+<head>
+  <meta charset="utf-8">
+  <title>Car Assistant</title>
+  <script src="lib/angular/angular.min.js"></script>
+  <script src="lib/angular-route/angular-route.min.js"></script>
+  <script src="lib/angular-resource/angular-resource.min.js"></script>
+  <script src="lib/bootstrap/dist/js/bootstrap.min.js"></script>
+  <script src="lib/ionic/js/ionic.bundle.js"></script>
 
-    <link rel="stylesheet" href="libs/ionic/css/ionic.css" >
-    <link rel="stylesheet" href="css/style.css">
-  </head>
+  <link rel="stylesheet" href="lib/ionic/css/ionic.css">
+  <link rel="stylesheet" href="src/cars/styles/local_style.css">
+  <link rel="stylesheet" href="src/apps/styles/global_style.css">
+  <link rel="icon" href="favicon.ico">
+</head>
 
-  <body ng-app="ionicApp" ng-controller="MainCtrl">
-    <ion-nav-view></ion-nav-view>
+<body ng-app="ionicApp" ng-controller="MainCtrl">
+<ion-nav-view></ion-nav-view>
 
-    <script src="js/app.js"></script>
-    <script src="js/app.route.js"></script>
-    <script src="js/app.controller.js"></script>
-    <script src="js/app.service.menu.js"></script>
-    <script src="js/car/car.module.js"></script>
-    <script src="js/car/car.service.js"></script>
-    <script src="js/car/car.controller.js"></script>
+<script src="src/apps/modules/app.js"></script>
+<script src="src/apps/routers/app.route.js"></script>
+<script src="src/apps/controllers/app.controller.js"></script>
+<script src="src/apps/services/app.service.menu.js"></script>
+<script src="src/cars/modules/car.module.js"></script>
```

Comment:
Why in  'module_name'  you need one more 'module' folder? Place your name.module.js file into a module root dir.